lib/DDG/Spice/SkyscannerFlightSearch.pm

```diff
@@ -0,0 +1,39 @@
+package DDG::Spice::SkyscannerFlightSearch;
+
+# ABSTRACT: This IA will call Skyscanner's API to retrieve the cheapest flight prices to 
+# the chosen destination
+
+use DDG::Spice;
+use strict;
+use warnings;
+
+# Caching - http://docs.duckduckhack.com/backend-reference/api-reference.html#caching
+spice is_cached => 1;
+spice proxy_cache_valid => '200 1d'; # defaults to this automatically
+
+spice wrap_jsonp_callback => 1; # only enable for non-JSONP APIs (i.e. no &callback= parameter)
+spice to => 'http://partners.api.skyscanner.net/apiservices/browseroutes/v1.0/GB/GBP/en-GB/$1/anywhere/anytime/anytime?apikey={{ENV{DDG_SPICE_SKYSCANNER_APIKEY}}}';
```

Comment:
To clarify, what I meant, you can `return` multiple values in the `handle` and for example if the locale is the 2nd value, you can replace `en-GB` with `$2`. You can pass along as many values as needed.

When you're passing 2+ values though, you need to define a `spice from` regex to capture them.

For two values, `spice from => "/([^\/]+)/([^\/]+)"` should work, for three, `spice from => "/([^\/]+)/([^\/]+)/([^\/]+)`, etc.

Should be lots of examples in the repo 👍